source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -743,8 +743,8 @@ def get(self, region=None, lon=None, lat=None, uri=None):
     def post(self, region=None, lon=None, lat=None, uri=None):
         args = self.parsers['post'].parse_args()
         if args['traveler_type']:
```

Comment:
use of direct dict access here, while use of dict.get() above (in the get() part). Is it intendend ?